src/db.coffee

```diff
@@ -180,15 +180,15 @@ module.exports = class DB
 							t.increments('id').primary()
 							t.string('uuid')
 							t.string('appId')
-							t.string('localId')
+							t.string('local_id')
 							t.string('device_type')
 							t.string('logs_channel')
 							t.string('deviceId')
 							t.boolean('is_online')
 							t.string('name')
 							t.string('status')
-							t.string('download_progress')
-							t.string('is_managed_by')
+							t.integer('download_progress')
+							t.string('device')
```

Comment:
Isn't this the same a s the `deviceId` above?

Reply:
https://github.com/resin-io/resin-api/blob/master/src/models/resin.sbvr#L502-L505 is the model. I incorrectly thought it would translate into a field called `is_managed_by` on the DB but it actually translates to `device` as a FK pointing to the managing device